pages/instructorIssues/instructorIssues.ejs

```diff
@@ -3,6 +3,11 @@
   <head>
     <%- include('../partials/head'); %>
     <link href="<%= asset_path('stylesheets/issues.css') %>" rel="stylesheet" />
+    <script>
+      $(() => {
+          $('[data-toggle=popover]').popover({sanitize: false, trigger: 'hover'});
```

Comment:
Wondering if a tooltip is better in this case? Not sure how this impacts accessibility. Tooltips are explicitly mentioned in the docs as being accessible via ARIA rules.

Reply:
A tooltip as implemented is still problematic, as the trigger cannot be focused by the keyboard. Check out the note on https://getbootstrap.com/docs/4.6/components/tooltips/#markup:

> You should only add tooltips to HTML elements that are traditionally keyboard-focusable and interactive (such as links or form controls). Although arbitrary HTML elements (such as <span>s) can be made focusable by adding the tabindex="0" attribute, this will add potentially annoying and confusing tab stops on non-interactive elements for keyboard users, and most assistive technologies currently do not announce the tooltip in this situation. Additionally, do not rely solely on hover as the trigger for your tooltip, as this will make your tooltips impossible to trigger for keyboard users.

Reply:
I changed it to a tabable a, is that a reasonable option?